data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,31 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4105508"
+    name = "pr_cianorte"
+    allowed_domains = ["cianorte.pr.gov.br"]
+    start_urls = ["https://cianorte.pr.gov.br/orgao-oficial/"]
+    current_page = 1
```

Comment:
We could avoid using `current_page` as a class attribute by using [`cb_kwargs`](https://docs.scrapy.org/en/latest/topics/request-response.html#passing-additional-data-to-callback-functions).
```python
    def parse(self, response, current_page=1):
        ...
                yield Request(self.base_url + str(current_page), cb_kwargs=current_page+1)
```